src/pages/docs/packaging-applications/build-servers/github-actions.md

```diff
@@ -1,557 +1,131 @@
-      OCTOPUS_URL: ${{ secrets.SERVER }}
-      OCTOPUS_SPACE: 'Outer Space'
-    with:
-      server_task_id: ${{ fromJson(steps.some_previous_deployment_step.outputs.server_tasks)[0].serverTaskId }}
-```
-
-### Create Nuget Package
+Octopus Deploy GitHub Actions can be run on every available type of [runners](https://docs.github.com/en/actions/about-github-actions/understanding-github-actions#runners) (Ubuntu Linux, Microsoft Windows, macOS, and Self-Hosted). 
 
-Create NuGet package(s) to push to Octopus Deploy.
+If your Octopus Server is not accessible over the internet, you can connect to it using a [Self-Hosted runner.](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners)
 
-```yaml
-steps:
-  - uses: actions/checkout@v3
-
-  # create a NuGet package from files in the "reports" folder; create package in "packaging" folder
-  - name: Create a NuGet package 🐙
-    uses: OctopusDeploy/create-nuget-package-action@v3
-    with:
-      package_id: 'DemoPackage'
-      version: '1.0.0'
-      output_folder: 'packaging'
-      base_path: reports
-      files: |
-        **/*.*
-      nuspec_description: package description
-      nuspec_authors: |
-        author 1
-        author 2
-      nuspec_release_notes: |
-        This is a multiline
-        release note
-```
 
-### Create Zip Package
+## Sequencing Tasks
 
-Create Zip package(s) to push to Octopus Deploy.
-
-```yaml
-steps:
-  - uses: actions/checkout@v3
-
-  # create a Zip package from files in the "reports" folder; create package in "packaging" folder
-  - name: Create a Zip package 🐙
-    uses: OctopusDeploy/create-zip-package-action@v3
-    with:
-      package_id: 'DemoPackage'
-      version: '1.0.0'
-      output_folder: './packaging'
-      base_path: reports
-      files: |
-        **/*.*
-```
+It can be useful to run multiple Octopus Deploy GitHub Actions in sequence as part of a larger workflow. To do this, simply include each Octopus Action as a step within a single job. 
 
-### Install Octopus CLI Action v3
+If you need to run sequential Octopus Actions in separate jobs, you can also configure your jobs to run sequentially by [defining prerequisite jobs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-jobs-in-a-workflow#defining-prerequisite-jobs).
 
-Although the CLI is not required for GitHub Actions v3, the install Octopus CLI Action v3 will now install the new [Octopus CLI](https://octopus.com/docs/octopus-rest-api/cli).
 
-:::div{.warning}
-If you require the use of the original Octo CLI, you can use `OctopusDeploy/install-octopus-cli-action@v1`.
-:::
+## Previous Versions
 
+Since the release of v3, Octopus Deploy GitHub Actions no longer need the [Install Octopus CLI](https://github.com/marketplace/actions/install-octopus-cli) package to be installed before running. Each Octopus Action introduced before v3, provides a guide to migrating to v3.
```

Comment:
```suggestion
Since the release of v3, Octopus Deploy GitHub Actions no longer need the [Install Octopus CLI](https://github.com/marketplace/actions/install-octopus-cli) package to be installed before running. Each Octopus Action introduced before v3 provides a guide to migrating to v3.
```